src/state/questionSlice.ts

```diff
@@ -0,0 +1,34 @@
+import { Action, createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+export interface QuestionState {
+  code?: string;
+  submittedAnswerTrue?: boolean;
+  answer?: number;
+  correctAnswer?: number;
+}
+
+export const defaultQuestionState: QuestionState = {};
+
+const questionStateSlice = createSlice({
+  name: "codeSlice",
+  initialState: defaultQuestionState,
+  reducers: {
+    setCode: (state: QuestionState, action: PayloadAction<string>): void => {
```

Comment:
What is the purpose of having setCode inside the questionSlice? I think this might be a relic from the gameSlice that we forgot to delete

Reply:
it was, I removed it